testing/run

```diff
@@ -144,15 +153,12 @@ fi
 
 if [[ -z "${1:-}" ]]; then
     echo
-    echo "FATAL: No entrypoint provided ..."
+    echo "WARNING: No entrypoint provided ... executing default osde2e tests"
```

Comment:
"... This warning can be ignored if you're seeing this in osde2e logs"

Reply:
👍🏽